docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc

```diff
@@ -125,16 +139,14 @@ quarkus.http.auth.permission.deny1.paths=/public/*
 quarkus.http.auth.permission.deny1.policy=deny
 ----
 
-TIP: Given the above permission set, `GET /public/foo` would match both permission sets' paths,
-but because it matches the `permit1` permission set's explicit method, `permit1` will be chosen and the request will
-be accepted. `PUT /public/foo` on the other hand, will not match the method permissions of `permit1` and so
-`deny1` will be activated and reject the request.
+TIP: Given the above permission set, `GET /public/foo` would match both permission sets' paths, but because it matches the `permit1` permission set's explicit method, `permit1` will be chosen, and the request will
```

Comment:
```suggestion
TIP: Given the above permission set, `GET /public/foo` would match the paths of both permission sets, but because it matches the explicit method of the `permit1` permission set, `permit1` will be chosen, and the request will
```